sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -132,6 +132,24 @@ struct _pi_device : _ur_device_handle_t {
   using _ur_device_handle_t::_ur_device_handle_t;
 };
 
+// Structure describing the command queue descriptor when command lists are
+// created. This is used to ensure that the appropriate command list type is
+// reused from the context's cache. Only immediate command lists are recycled
+// across queues and then all fields are used. For standard command lists only
```

Comment:
> Only immediate command lists are recycled across queues

This is not true. Regular command-lists are re-used anywhere in context. It's just that they are additionally "connected" by fences to specific command-queues, and thus do not use this queue descriptor (except for ordinal I believe)